frida_mode/test/fasan/GNUmakefile

```diff
@@ -0,0 +1,99 @@
+PWD:=$(shell pwd)/
+ROOT:=$(shell realpath $(PWD)../../..)/
+BUILD_DIR:=$(PWD)build/
+LIBASAN:=/usr/lib/llvm-10/lib/clang/10.0.0/lib/linux/libclang_rt.asan-x86_64.so
```

Comment:
The hard-coded asan path could be an issue, this should probably use `llvm-config` to get the path (or something?)

Reply:
yes that will break on so many platforms. needs llvm-config (and require LLVM_CONFIG env set otherwise)

Reply:
Sure. No problems, do you have an example I can use as a reference? No worries if not, I’ll fire up my Google.

Reply:
just copy-paste what is done in GNUmakefile.llvm to get a working llvm-config